jetty-distribution/src/main/resources/etc/fileserver.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure.dtd">
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+
+  <Set name="handler">
+    <New class="org.eclipse.jetty.server.handler.HandlerList">
```

Comment:
Have a look a the server module and you will see that it uses a `jetty.xml` file.  Look at that and you will see that it creates the handler list and also a `ContextHandlerCollection` called "Contexts", so your xml should create a ContextHandler which it adds to Contexts and then the resource handler should be part of that.

So it will be something like:
```xml
<Ref refid="Contexts">
  <Call name="addHandler">
    <New class="org.eclipse.jetty.handler.ContextHandler">
       <Set name="contextPath">/</Set>
       <Call name="addHandler">
         <New class="o....ResourceHandler/>
       
          etc.
          etc.
```